test/api/v1/routes/exercises_test.rb

```diff
@@ -63,6 +63,28 @@ def test_restore_exercises_and_solutions
     end
   end
 
+  def test_get_exercises_by_track
+    VCR.use_cassette('exercism_api_error') do
```

Comment:
I don't think you need to use a cassette here since you're halting before the API call is made due to the track not being found. Does this test past locally if you remove this?